.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+
+notifications:
+  email: false
+
+before_install:
+  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes
+  - conda update conda
+  - conda install conda-build
+  - conda config --add channels omnia
+  - conda info -a
+
+install:
+  - conda create --name test-environment python=$TRAVIS_PYTHON_VERSION pyemma msmbuilder
+  - source activate test-environment
+  - conda build devtools/conda-recipe
```

Comment:
This doesn't actually install any of your dependencies in the `test-environment` before calling `python setup.py install`, so everything will fail.

I'd highly recommend using the idiom we've settled on:
```YAML
script:
  # Add omnia channel
  - conda config --add channels ${ORGNAME}
  # Add omnia dev channel
  - conda config --add channels https://conda.anaconda.org/omnia/label/dev
  # build the recipe
  - conda build devtools/conda-recipe
  # Test the local installation
  - source activate _test
  # Install OpenEye toolkit
  - pip install $OPENEYE_CHANNEL openeye-toolkits && python -c "import openeye; print(openeye.__version__)"
  # Run tests
  - cd devtools && nosetests $PACKAGENAME --nocapture --verbosity=2 --with-doctest --with-timer && cd ..
```